<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowForm</class>
 <widget class="QWidget" name="WindowForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BioCheat 0.9.6</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="capBox">
     <property name="title">
      <string>Rectangle (Capture)</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="xOffsetLabel">
        <property name="text">
         <string>Left</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="left">
        <property name="suffix">
         <string> px</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="regionWidthLabel">
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="width">
        <property name="suffix">
         <string> px</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="yOffsetLabel">
        <property name="text">
         <string>Top</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="top">
        <property name="suffix">
         <string> px</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="regionHeightLabel">
        <property name="text">
         <string>Height</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="height">
        <property name="suffix">
         <string> px</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="yOffsetLabel_2">
        <property name="text">
         <string>Frequency</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="frequency">
        <property name="suffix">
         <string> Hz</string>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QCheckBox" name="onTop">
        <property name="text">
         <string>Always on top</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>AI Tuning (Recognizer and Hinter)</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="xOffsetLabel_2">
        <property name="text">
         <string>Hor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="hBlocks">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="regionWidthLabel_2">
        <property name="text">
         <string>Vert</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="vBlocks">
        <property name="suffix">
         <string/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="regionWidthLabel_3">
        <property name="text">
         <string>Valid</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QSpinBox" name="valid">
        <property name="suffix">
         <string/>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="xOffsetLabel_3">
        <property name="text">
         <string>Sens</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="sensitivity">
        <property name="minimum">
         <number>70</number>
        </property>
        <property name="value">
         <number>90</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QCheckBox" name="highlight">
        <property name="toolTip">
         <string>If there are 4-combos or 5-combos, show only that moves and hide the others.</string>
        </property>
        <property name="text">
         <string>Best Moves</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="4" colspan="2">
       <widget class="QCheckBox" name="apBest">
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;If &lt;span style=&quot; font-weight:600;&quot;&gt;selected&lt;/span&gt;, Autoplay using the best move only. If &lt;span style=&quot; font-weight:600;&quot;&gt;unselected&lt;/span&gt;, use a random move.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Best AP</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QRadioButton" name="display0">
        <property name="text">
         <string>&amp;Off</string>
        </property>
        <property name="shortcut">
         <string>Alt+O</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="display2">
        <property name="text">
         <string>&amp;Recognize</string>
        </property>
        <property name="shortcut">
         <string>Alt+R</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="display3">
        <property name="text">
         <string>&amp;Hints</string>
        </property>
        <property name="shortcut">
         <string>Alt+H</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QRadioButton" name="display4">
        <property name="text">
         <string>&amp;AutoPlay!</string>
        </property>
        <property name="shortcut">
         <string>Alt+A</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="Visualizer" name="visualizer">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="display1">
        <property name="text">
         <string>&amp;Capture</string>
        </property>
        <property name="shortcut">
         <string>Alt+C</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="5"/>
 <customwidgets>
  <customwidget>
   <class>Visualizer</class>
   <extends>QFrame</extends>
   <header location="global">Visualizer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>display1</tabstop>
  <tabstop>display2</tabstop>
  <tabstop>display3</tabstop>
  <tabstop>display4</tabstop>
  <tabstop>display0</tabstop>
  <tabstop>capBox</tabstop>
  <tabstop>left</tabstop>
  <tabstop>width</tabstop>
  <tabstop>top</tabstop>
  <tabstop>height</tabstop>
  <tabstop>frequency</tabstop>
  <tabstop>onTop</tabstop>
  <tabstop>groupBox</tabstop>
  <tabstop>hBlocks</tabstop>
  <tabstop>vBlocks</tabstop>
  <tabstop>valid</tabstop>
  <tabstop>sensitivity</tabstop>
  <tabstop>highlight</tabstop>
  <tabstop>apBest</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
